examples/let-expression/let_expression.bal

```diff
@@ -0,0 +1,47 @@
+import ballerina/io;
+
+public function main() {
+
+    // Following is a simple let expression which involves only a single
```

Comment:
```suggestion
    // This is a simple let expression which involves only a single
```

Since the comments are rendered on the side atm.

Reply:
+1, changed.